Source/MonoGame.Extended.Tests/AngleTest.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using Microsoft.Xna.Framework;
+using NUnit.Framework;
+
+namespace MonoGame.Extended.Tests
+{
+    [TestFixture]
+    public class AngleTest
+    {
+        private const float DELTA = 0.0000001f;
```

Comment:
That was a naming convention we used at school, so it should be `_delta` for private constants and `Delta` for public ones from now on?